DQM/Integration/python/clients/l1tstage2_dqm_sourceclient-live_cfg.py

```diff
@@ -8,6 +8,8 @@
 # Live Online DQM in P5
 process.load("DQM.Integration.config.inputsource_cfi")
 process.load("DQM.Integration.config.FrontierCondition_GT_cfi")
+process.source.SelectEvents = cms.untracked.vstring('HLT_L1FatEvents_v*')
```

Comment:
This is the line to comment in order to turn off the HLT fat event filtering for L1T DQM!